src/track/track.cpp

```diff
@@ -1451,8 +1451,7 @@ mixxx::track::io::key::ChromaticKey Track::getKey() const {
 
 // returns the formatted key for display purpose
 QString Track::getKeyText() const {
-    const auto locked = lockMutex(&m_qMutex);
-    return m_record.getKeys().getGlobalKeyText();
+    return KeyUtils::keyToString(getKey());
```

Comment:
this doesn't return the users (untouched) keytext anymore, isn't that a regression?

Reply:
I think this was a bug before. Before i think it has returned the key in the format the was selected when the track was analyzed. Since this is used in the library it should reflect the current selected. Did you found an issue wit that?

Reply:
Well it seems weird because from what I can tell is that the `get*KeyText()`-like methods return the original string while the `get*Key()` methods return the info from the analyzer. This assumption breaks here since we're returning the value from `getKey()`. Does that make sense?

Reply:
I have double checked it. `Track::getKeyText()` is only used for searching and "find related". Before  m_record.getKeys().getGlobalKeyText(); returned the preferences formatted String. This has been changed to the unformated Key form the metadata. The change was required to have here still the preferences formatted string. 

In case the analyzer finds a different key "m_record.getKeys().getGlobalKeyText();" will become nor the ID3v2 formatted key text.